packages/console/src/pages/SignInExperience/components/Preview.tsx

```diff
@@ -197,7 +195,18 @@ const Preview = ({ signInExperience, className }: Props) => {
                 <PhoneInfo />
               </div>
             )}
-            <iframe ref={previewRef} src="/sign-in?preview=true" tabIndex={-1} />
+            {
+              // We set both "allow-scripts" and "allow-same-origin" to allow communication with the iframe
+              // This is intended since the source is trusted
+              /* eslint-disable react/iframe-missing-sandbox */
```

Comment:
Do we need to change the rule to allow scripts?

Reply:
<img width="727" alt="image" src="https://user-images.githubusercontent.com/12833674/190097785-3cfd3cec-343a-4774-9c8d-b0eb7c310d61.png">
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe#attr-sandbox

Looks good to me as the Mozilla documentation recommended this way

Reply:
if so, why don't we just disable the rule and remove the sandbox attribute lol

Reply:
you're right, for trusted content, we can just remove the attribute

Reply:
updated

Reply:
Well, I thought you didn't want to disable the rule but fix it instead, as I saw the giant "FIXME" on top of the file.
Indeed, both ways are fine.

Reply:
haha, but it turns out that we can't fix it.